config/babel/babel.config.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
   ],
   presets: [
     ['@babel/preset-env', {
-      corejs: '3.10',
+      corejs: '3.12.1',
```

Comment:
https://babeljs.io/docs/en/babel-preset-env#corejs

```suggestion
      corejs: '3.12',
```